### DISCUSSION TITLE
Where did the nameservers documentation go?

### DISCUSSION THREAD

**Discussion by stephenhmarsh** [vercel/community#418] [Help] [answered]
The documentation still refers to a "nameservers method" for configuring a domain, but there is no documentation about it.

Specifically, **I really need the list of Vercel's nameservers** to set at a client's registrar.

I remember them being something like:

```
ns1.vercel-dns.com
ns2.vercel-dns.com
... etc.
```

but now I have no idea if those still even work??

Why??????

Help

**Comment by FreshP0325** [comment, marked as answer]
That's it. The present nameservers are 

```
ns1.vercel-dns.com
ns2.vercel-dns.com
```

or 

```
ns1.vercel-dns-2.com
ns2.vercel-dns-2.com
```

As far as I am concerned, it is why the former nameservers are in this format that consfuses  you, like these,

```
a.zeit-world.org
b.zeit-world.net
c.zeit-world.com
d.zeit-world.co.uk
```

In brief, however, they do play a same role in resolving your domain name, and the brand-new one might be faster due to the decrease in the amount of nameservers, reducing the traffic your visitors cost.

You can check your domain names' nameservers after you associating one sub-domain or apex domain to your project on Vercel at the project's Settings > Domains >  Domains' Status.

Hope this helps.

> a little tip that appears to be useless - Vercel DNS perviously partnered with Constellix.com while now is NS1.com

**Reply by paulogdm** [reply]
**EDIT** Use:
```
ns1.vercel-dns.com
ns2.vercel-dns.com
```
===
Original post:
A clarification. The following are perfectly valid still:
```
a.zeit-world.org
b.zeit-world.net
c.zeit-world.com
d.zeit-world.co.uk
```
However, we do recommend that you use the new ones:
```
ns1.vercel-dns.com
ns2.vercel-dns.com
```

**Comment by benoj** [comment]
This is very useful - only place I can find this on the web - this should be in your docs somewhere

**Comment by xndyz** [comment]
Looks like it's been added to the docs (although could be more conspicuous) https://vercel.com/docs/custom-domains#vercel-nameservers

**Comment by made-by-chris** [comment]
this was impossible to find and made me wayyyy overthink this process :D should be the first line in your custom domains section.

**Reply by leerob** [reply]
To clarify, what did you search to end up here and where would you have expected to find it? I can update the docs. And what specifically were you looking for?

```
ns1.vercel-dns.com
ns2.vercel-dns.com
```

**Reply by made-by-chris** [reply]
My expectation based on my modest knowledge of buying, setting up, "pointing" domains to the service where im using them is "just gimme the nameservers so i can redirect" and that's typically the extent of all i do with domains. So anywhere on the first page (in this case https://vercel.com/docs/custom-domains ) a mention of the nameservers would be appreciated. I went down a rabbit hole to end up on this page, im sorry i cant remember exactly what i googled in the end. None of the information, as far as I can tell, on the vercel domains documentation page was applicable to my use-case, and it led me to believe there were no nameservers, and i had to do something more manual/low-level to get it working. I dunno how typical my use-case was, but i imagine its fairly common. "Push my app to vercel, stick a domain on it". 
When i google "vercel nameserver" i expect to see the nameservers on the first ( or any of the first ) result.

**Reply by benoj** [reply]
I can't remember how I got here as it was a long time ago - but I do remember trawling various options and out of date SO comments etc..

I would probably have something simple like the following on the custom-domains page:

**Want to use our nameservers but bought your domain somewhere else?**

Update your NS records to :

ns1.vercel-dns.com
ns2.vercel-dns.com

**Reply by made-by-chris** [reply]
oh interesting - i looked through vercel site to buy a domain directly and couldn't find that either :P is that a feature somewhere?

**Reply by jarvisjohnson** [reply]
Yeah experiencing this now - have been unable to get my site live for over an hour, frustrating.

In the [docs](https://vercel.com/docs/custom-domains#vercel-nameservers) linked above it shows:

![image](https://user-images.githubusercontent.com/12746936/136645053-ba5b7f61-c566-4908-ac03-e58d4b6c6130.png)


But the "nameservers" option just doesn't actual exist in there as far as I can tell, there is only the A record / CNAME option, which I can't get to work (using google domains nameservers).  Literally everything else with vercel and nextjs is so flawless except this absolutely critical step of putting the site live 😅


Will update when I figure it out...

**Reply by bitfede** [reply]
Experiencing the same issue... Not sure where to find a spot in the UI that gives me the nameservers.

I will try with

```
ns1.vercel-dns.com
ns2.vercel-dns.com
```

**Reply by jarvisjohnson** [reply]
Yep those are the right nameservers 👍

**Reply by m-torin** [reply]
Thanks! Vercel still does not have this documented.

**Comment by shonsirsha** [comment]
Really, about time to add this in the docs.
